.p4-env.json

```diff
@@ -11,6 +11,20 @@
     },
     "composer": {
       "processTimeout": "600"
+    },
```

Comment:
Adding `"xdebug.client_host": "host.docker.internal"` to make it work the debugger on MacOs.